src/LightGBM.jl

```diff
@@ -16,21 +16,25 @@ struct LibraryNotFoundError <: Exception
 end
 
 
-function find_library(library_name::String, custom_paths::Vector{String})
+function find_library(library_names::Vector{String}, custom_paths::Vector{String})
```

Comment:
Theres no need to make this to a `Vector{String}` (because we're not hunting for multiple libs).

As I understand, the "fix" here is to call `Libdl.find_library` like this: `Libdl.find_library([library_name])` instead of `Libdl.find_library(library_name)`